processor/processor.go

```diff
@@ -114,54 +174,52 @@ func (p *Processor) isError(e ast.Field) bool {
 
 func (p *Processor) Process(fset *token.FileSet, file *ast.File) error {
 	var patches []patch
+	var funcLitCond *funcTypeConditions
 
 	astutil.Apply(file, nil, func(c *astutil.Cursor) bool {
 		if c == nil {
 			return true
 		}
 
-		fn, ok := c.Node().(*ast.FuncDecl)
-		if !ok || fn == nil {
-			return true
-		}
+		p.packageName(c)
 
-		fname := p.functionName(*fn)
-		if !p.FunctionSelector.AcceptFunction(fname) {
-			return true
-		}
+		switch c.Node().(type) {
+		// original
+		case *ast.FuncDecl:
```

Comment:
this case in switch probably need to delegate to func. try keep same signature as with p.processAnonymousFunction
```suggestion
		case *ast.FuncDecl:
		    p.processFunction(c.Node().(*ast.FuncLit))
```